src/main/java/model/piece/Piece.java

```diff
@@ -0,0 +1,29 @@
+package model.piece;
+
+import java.util.Map;
+import model.Path;
+import model.Point;
+import model.Team;
+
+public abstract class Piece {
+    Team team;
+    PieceName pieceName;
+
+    public Piece(Team team) {
```

Comment:
추상 클래스는 생성자 접근 범위를 제한해주세요!

```suggestion
    protected Piece(Team team) {
```

Reply:
Public 으로 열어두게 된다면, 오버라이딩을 하게끔 뜨던데 protected로 사용하는게 원래의 의도같네요!

감사합니다 😀